tests/src/runner/podman-desktop-runner.ts

```diff
@@ -51,29 +51,50 @@ export class PodmanDesktopRunner {
     if (this.isRunning()) {
       throw Error('Podman Desktop is already running');
     }
-
-    // start the app with given properties
-    this._app = await electron.launch({
-      ...this._options,
-    });
-    // setup state
-    this._running = true;
-    this._page = await this.getElectronApp().firstWindow();
+    let retry = true;
```

Comment:
I would prefer this kind of changes to be done outside of a PR introducing new tests.

Reply:
reverted this part, will make another PR for this